### ISSUE TITLE
Pulltorefresh doesnt work

### ISSUE THREAD

**Issue by thunderforces** [mgks/Android-SmartWebView#117]
Pulltorefresh doesnt work very well . After activating it , 

there is no way to scrool up . You can go down but if you go back to up side of the screen , page is loaded again  and again

**Comment by thunderforces** [comment]
here you can find a video , I have prepared it to show you what happens . 
 
https://gofile.io/?c=4iQ8QQ

**Comment by mgks** [comment]
can you give some information on what android APIs you're working on.

**Comment by thunderforces** [comment]
Hi again 

I have uploaded all files to ;

https://gofile.io/?c=hoLPe2 

this way will be better to understand issue , you can download and try to run it .

thank you very much

**Comment by djmike3404** [comment]
I agree with this completely. Scrolling back up on a page is a nightmare when using scroll to refresh. Im on API 29. I just deactivated it as it crippled the app when using it
